docmost/docker-compose.yml

```
@@ -0,0 +1,41 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: docmost_web_1
+      APP_PORT: 3000
+
+  web:
+    image: docmost/docmost:0.20.4@sha256:f16f2a69425de9e0d026b00b4d1b3416b2b75ffa24003d43d6a181d76c52a352
+    user: "1000:1000"
+    depends_on:
+      - db
+      - redis
+    environment:
+      APP_URL: "http://${DEVICE_DOMAIN_NAME}:8941"
+      APP_SECRET: "QpFvscqyQeR7otMy6Eb6GttDYAGGc6WQCq4zY0HMmIfrXXqO0Ada3zwWi3Ic7U2K"
+      DATABASE_URL: 'postgresql://docmostuser:docmostpass@db:5432/docmost?sslmode=disable'
+      REDIS_URL: "redis://redis:6379"

```

Use of full service names.
```suggestion
      DATABASE_URL: 'postgresql://docmostuser:docmostpass@docmost_db_1:5432/docmost?sslmode=disable'
      REDIS_URL: "redis://docmost_redis_1:6379"
```

---

Done! 💪🏻